### ISSUE TITLE
Product always posted in English

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#513]
New products are always posted in English, even if the UX is in French.

```
5 mars 2017 à 19:09:12 CET - Images sélectionnées (Ajout : front_en) - new image nutrition_en : 2.4 - voir
5 mars 2017 à 19:09:12 CET - openfoodfacts-contributors Images téléchargées (Ajout : 2) - new image 2 - image upload - voir
5 mars 2017 à 19:09:12 CET - openfoodfacts-contributors Images téléchargées (Ajout : 1) - new image 1 - image upload - voir
5 mars 2017 à 19:09:06 CET - sebleouf Informations (Ajout : lang, product_name, quantity, countries, nutrition_data_per) - (app)new android app - voir```

**Comment by itchix** [comment]
Fixed with d134f7a90c9884c1e400984ad1f5c69d78f576d5

**Comment by teolemon** [comment]
Has it been deployed yet ? 
poke @itchix (reported by @tacite)
